class.phpmailer.php

```diff
@@ -1815,9 +1816,27 @@ public function setLanguage($langcode = 'en', $lang_path = '')
             if (!self::isPermittedPath($lang_file) or !is_readable($lang_file)) {
                 $foundlang = false;
             } else {
-                // Overwrite language-specific strings.
-                // This way we'll never have missing translation keys.
-                $foundlang = include $lang_file;
+                //$foundlang = include $lang_file;
+                $lines = file($lang_file);
+                foreach ($lines as $line) {
+                    //Translation file lines look like this:
+                    //$PHPMAILER_LANG['authenticate'] = 'SMTP-Fehler: Authentifizierung fehlgeschlagen.';
+                    //These files are parsed as text and not PHP so as to avoid the possibility of code injection
+                    //See https://blog.stevenlevithan.com/archives/match-quoted-string
+                    $matches = [];
```

Comment:
```suggestion
                    $matches = array();
```
This adaptation of the backport is needed for PHP 5.6 compatibility.

Reply:
This PHP 5.6 incompatibility remark has been ignored!

How can this review be ignored and the PR still be merged ?

Reply:
Ok, looks like my "fault". I missed the top right "Finish review" button, so looks that my review and remark was visible only to me!  :-(

Reply:
Ok, looks like this got fixed independantly in the joomla-backports branch.

Can be marked as solved.